src/adaptix/_internal/morphing/generic_provider.py

```diff
@@ -418,6 +433,19 @@ def union_dumper(data):
 
         return union_dumper
 
+    def _get_dumper_for_literal(
+        self,
+        dumper_type_dispatcher: ClassDispatcher[Any, Dumper],
+        literal_dumper: Any,
```

Comment:
You should use type hint ``Dumper`` here

Reply:
done